server/functions/ai_objectives/fn_ai_obj_request_pursuit.sqf

```diff
@@ -70,8 +70,26 @@ _objective setVariable ["onTick", {
 	// while other targets nearby are active.
 	private _targetIdx = _targets findIf {!(_x getVariable ["vn_revive_incapacitated", false])};
 
-	// selects the last target in the targets array if everyone is incapacitated
-	// TODO: patrol around or move away?
+	// everyone is incapacitated, let's patrol the area or set up an ambush
```

Comment:
Add a note here on what happens when we switch back from all players incap to >1 players incap (lines https://github.com/Bro-Nation/Paradigm/pull/32/files#diff-a11e1a39c5b7ba209e8ab4bacfea1c9a9ae6fdf3cb33606789aafabb869f7660R104-R107 kick in again, resetting the group orders)